pyaedt/modules/solutions.py

```diff
@@ -2786,6 +2872,22 @@ def field_plot_settings(self):
             ],
         ]
 
+    @property
+    def field_line_trace_plot_settings(self):
+        """Field line trace plot settings.
```

Comment:
Can this description (and the description for what is returned) be rewritten. It has four adjectives in front of the noun "settings".  Maybe "Settings for the trace plot of the field line."

Reply:
sure! thanks for feedback!